Plugins/ExternalTexture/Source/ExternalTexture_Shared.h

```diff
@@ -64,13 +66,17 @@ namespace Babylon::Plugins
         auto deferred{Napi::Promise::Deferred::New(env)};
         auto promise{deferred.Promise()};
 
+        DEBUG_TRACE("ExternalTexture AddToContextAsync");
```

Comment:
It would be good to add `this` pointer output to all of these traces so that they can be associated with each other.

Reply:
fixed

Reply:
Fixed.